subcommand/injector/command.go

```diff
@@ -206,11 +211,28 @@ func (c *Command) Run(args []string) int {
 	}
 
 	var handler http.Handler = mux
+	minTLSVersion, ok := tlsutil.TLSLookup[c.flagTLSMinVersion]
+	if !ok {
+		c.UI.Error(fmt.Sprintf("Failed to parse minimum TLS version %q", c.flagTLSMinVersion))
```

Comment:
The error message mentions parsing, but really we received an unsupported TLS version.